src/api/signals/revision.py

```diff
@@ -39,12 +39,14 @@ def _create_project_profile_revisions(query_kwargs):
 
 
 @receiver(pre_delete, sender=ProjectProfile)
-def update_delete_project_profile_revisions(sender, instance, *args, **kwargs):
+def delete_project_profile_revisions(sender, instance, *args, **kwargs):
```

Comment:
I think maybe it's worth a comment somewhere in this file explaining `related_to_profile_id` and noting that it is relevant to 1) deleting project profiles and 2) transferring CR ownership, for `pull/`s by the relevant user. And noting that 2) is handled in `replace.py` not by a signal.

Also: I'm sure it's here somewhere because it's previous functionality, but I'm not seeing how users *other than* `related_to_profile_id` know a project profile has been deleted?

Reply:
1. Added to note readme explaining "removes" rather than comments throughout the code

2. Other's receive a "delete" revision b/c the DB trigger creates one